modules/client/src/controllers/ConditionalTransferController.ts

```diff
@@ -69,22 +78,24 @@ export class ConditionalTransferController extends AbstractController {
       invalidXpub(recipient),
     );
 
-    const linkedHash = createLinkedHash(amount, assetId, paymentId, preImage);
+    // set recipient and encrypted pre-image on linked transfer
+    // TODO: use app path instead?
```

Comment:
Theres no great way to allow the recipient and sender to understand what the app key should be, since they don't know what nonce the other would be at or be expecting. I think leaving it as the free balance addr is best